Hospital.Generator.Nats.Host/Program.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Hospital.Generator.Nats.Host;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+var builder = Host.CreateApplicationBuilder(args);
+
+
+builder.Configuration
+    .AddJsonFile("appsettings.json", optional: true)
+    .AddEnvironmentVariables();
+
+builder.Services.Configure<NatsOptions>(
+    builder.Configuration.GetSection("Nats"));
```

Comment:
Лучше регистрировать через AddOptions
```suggestion
builder.Services.AddOptions<NatsOptions>()
    .Bind(builder.Configuration.GetSection("Nats"));
```